### ISSUE TITLE
Should be a way to vote continuously

### ISSUE THREAD

**Issue by vporton** [gitcoinco/passport#2138]
### User Story:
Votings in every Gitcoin round are independent. Because of this, it works well, when users are able to move (by canceling in the first and re-applying stamps in the second) stamps from one account to another one.

I need "continuous" voting: Users able to vote at any time, without voting split into rounds.

Therefore, for my Scorer ID, I need to workaround of moving stamps from one account to another one, because it would allow Sybil attack.

My proposed solution (if so chosen for a Scorer ID by the creator of the Scorer ID):

1. disable turning off stamps;
2. disable the old account as soon as one of its stamps is used in the new account.

Note that in this case need to exclude "unsafe" stamps (that could be too easily stolen by a third party).

**Comment by vporton** [comment]
"Too easily stolen" is a too flexible concept, therefore we need to allow the creator of a Scorer ID to choose which kinds of stamps to enable and which not.

To ease the creator of a Scorer ID job, let us mark every stamp kind with a number denoting its difficultly of being stolen. For instance, difficulty >10 could mean that it is prevented from being stolen by a decentralized authentication and authorization.

**Comment by nutrina** [comment]
I am trying to think if an alternative solution to what you proposed: would it work if a user could indicate that he owns multiple accounts (basically grouping all his accounts together), and we always consider any stamp that a user owns in any of his grouped accounts?

**Comment by vporton** [comment]
> I am trying to think if an alternative solution to what you proposed: would it work if a user could indicate that he owns multiple accounts (basically grouping all his accounts together), and we always consider any stamp that a user owns in any of his grouped accounts?

It would be not help to solve my problem, because it is entirely about multiple accounts, not about multiple addresses in a single account.

Above I propose to disable the entire user's account (if it is multi-address, then disable multiple addresses) as soon as a duplicate stamp is minted in another account (whether it's single-address or multi-address).

**Comment by vporton** [comment]
> The stamps are not actually bound to an account, they are bound to an ETH address

Yes, but **you** proposed to change it to have several addresses per account. I was saying under your assumption.

By account I mean a set of stamps of the same user.
